src/main/resources/application-dev.properties

```diff
@@ -16,19 +16,36 @@ ccd.data.store.api.url=http://localhost:4452
 tornado.url=https://eu.dws3.docmosis.com/api/render
 # tornado-docker
 #tornado.url=http://localhost:9095/api/render
-tornado.accessKey=${TORNADO_ACCESS_KEY:}
+tornado.accessKey=${TORNADO_ACCESS_KEY:ZDQ0ZWNjYWUtZGE2Yi00OWQ4LThlNDAtYjBhZmY4MDViZWJlOjM0MTM4ODc}
```

Comment:
No, we need to rotate these keys immediately

Reply:
done